jobs/store/templates/bin/nginx

```diff
@@ -4,13 +4,15 @@ set -eu
 RUN_DIR=/var/vcap/sys/run/store
 LOG_DIR=/var/vcap/sys/log/store
 JOB_DIR=/var/vcap/jobs/store
+DAT_DIR=/var/vcap/store/webdav
```

Comment:
Is this change related?

Reply:
is a fix that is already in master
https://github.com/starkandwayne/shield-boshrelease/commit/82894b42c65c2d7ea3cb25d54d9b56e7c5c7a6fb

the store job was not working at all without that fix